removal-waitlist.js

```diff
@@ -7,6 +7,12 @@ async function getHashedWaitlist() {
   return hashedWaitlistArray.toString().split("\n");
 }
 
+async function getHashedAdminList() {
+  const hashedAdminList = await readFile("hashed-admin-list.txt", "binary");
+  return hashedAdminList.toString().split("\n");
```

Comment:
If we pull the admin hashes from an env var, this code can become:
```suggestion
  const hashedAdminList = AppConstants.REMOVAL_ADMINS;
  return hashedAdminList.toString().split(",");
```

Reply:
Though now that I think about, if we put them list of admins in an env var, there's no need to hash them anymore either.